src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Conditions/Condition.cs

```diff
@@ -15,29 +15,29 @@ public enum Condition
     {
         [DisplayName("Contains")]
         Contains,
-        [DisplayName("StartsWith")]
```

Comment:
I know, but Catel do not humanize it like Ben wanted. Catel just returns display names.